cmc/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for the HMCTS CMC product
 name: cmc
-version: 2.0.0
+version: 2.0.1
```

Comment:
this doesn't actually need changing, it will just use whatever is in the git tag

Reply:
ok will revert